go/stellar/stellarsvc/frontend.go

```diff
@@ -1251,3 +1251,11 @@ func (s *Server) GetStaticConfigLocal(ctx context.Context) (res stellar1.StaticC
 		PublicMemoMaxLength:  libkb.MaxStellarPaymentPublicNoteLength,
 	}, nil
 }
+
+func (s *Server) AssetDepositLocal(ctx context.Context, arg stellar1.AssetDepositLocalArg) (stellar1.AssetActionResultLocal, error) {
+	return stellar1.AssetActionResultLocal{}, errors.New("nyi")
```

Comment:
`nyi`?

Reply:
not yet in?

Reply:
not yet implemented....

i'm about to implement it.

Reply:
(I have a branch off of this one with the implementation started, but it could be advantageous to have this in so someone else could do some of the frontend work concurrently)